backend/clickhouse/logs.go

```diff
@@ -148,6 +148,7 @@ func (client *Client) LogsKeyValues(ctx context.Context, projectID int, keyName
 	query := sq.Select("LogAttributes[?] as value, count() as cnt").
 		From("logs").
 		Where(sq.Eq{"ProjectId": projectID}).
+		Where("mapContains(LogAttributes, ?)", keyName).
```

Comment:
:tada: TIL that you can do `WHERE mapContains(LogAttributes, 'foo')` and clickhouse/sql will check that it is truthy, without you having to explicitly say `WHERE mapContains(LogAttributes, ?) = 1`